assignments/this.js

```diff
@@ -13,14 +13,75 @@
 
 // code example for Window Binding
 
+
+var sayAge = function(){
```

Comment:
Were you able to see the window object when you ran it like this? I played around with it to see it, hope you don't mind :-)


  var sayAge = function(){
    // 'use strict';
    console.log(this);
};

var me = {
    age: 26
};

sayAge();